docker/docker-compose.yml

```diff
@@ -65,20 +66,20 @@ services:
 
       # =====================================
       # Web Concurrency
-      WEB_GUNICORN: true
+      WEB_GUNICORN: "true"
```

Comment:
This is the only change I think is warranted. 

This docker-compose file is meant for end users so the annotations aren't super useful. For end-users you should be using the documentation. This also doesn't seem to be a problem on newer versions of docker-compose? 🤷 

- https://nightly.mealie.io/documentation/getting-started/installation/postgres/

Configuration is also documented 

- https://nightly.mealie.io/documentation/getting-started/installation/backend-config/
- https://nightly.mealie.io/documentation/getting-started/installation/frontend-config/